### ISSUE TITLE
twitter-style outputs year for some same-year timestamps

### ISSUE THREAD

**Issue by hossameldeen** [catamphetamine/javascript-time-ago#24]
```typescript
import TimeAgo from 'javascript-time-ago';
import en from 'javascript-time-ago/locale/en';

TimeAgo.addLocale(en);
const jsTimeAgo = new TimeAgo('en-US');
console.log(jsTimeAgo.format(1555549347000, 'twitter'));
```

**Expected:** Apr 18.

**Actual:** Apr 18, 2019.

**Note:** It doesn't shows the year for all same-year dates, though. For example, it correctly doesn't show the year for `Oct 18` and `Oct 18`.

**Version:** 2.0.1

**Screenshot:**
![image](https://user-images.githubusercontent.com/6719916/67138183-f774d000-f23f-11e9-9ab9-325bfe56eaf6.png)

**Comment by catamphetamine** [comment]
Released `javascript-time-ago@2.0.3`.

**Comment by hossameldeen** [comment]
Thank you :+1: 

Haven't tried it yet, but will tell you if there's a problem.
